Neolution.DotNet.Console/Neolution.DotNet.Console.csproj

```diff
@@ -1,33 +1,34 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
```

Comment:
is it still ok to target net6.0?
It will not force the project to upgrade due the include of the microsoft.extensions.* packages with version 8?

Reply:
Yes, because it should make the transition easier if the tool (that is getting updated) references .NET 6 SDK. 
As far as I've seen it, you will still forced to update Microsoft.Extensions.* packages to version 8.* if an update is required based on the dependencies of the tool project.